Document/Document.ipynb

```diff
@@ -46,6 +46,7 @@
     "    <li><a href=\"#Usage\">Usage</a></li>\n",
     "        <ol>\n",
     "            <li><a href=\"#From-vector\">From Vector</a></li>\n",
+    "            <li><a href=\"#From-array\">From Array</a></li>\n",
```

Comment:
Is it necessary to create a new section?
It's not wrong, but actually, it's some kind of `Direct CM`!